docs/development.rst

```diff
@@ -398,6 +398,81 @@ Common resources that a test might need,
 like e.g. a running MSColab server or a QApplication instance for GUI tests,
 are collected in :mod:`tests.fixtures` in the form of pytest fixtures that can be requested as needed in tests.
 
+Testing Keyring Features
+=========================
+
+This document outlines step-by-step instructions for testing keyring features using the command line.
+
+Prerequisites
+-------------
+
+1. **Confirm the Default Keyring Backend**
+
+   Use the following command to list available keyring backends and check which one is currently in use:
+
+   .. code-block:: bash
+
+       keyring --list-backends
+
+Command-Line Commands for Keyring
+-----------------------------------
+
+1. **Set a Password**
+
+   Store a password for a specific service and user:
+
+   .. code-block:: bash
+
+       keyring set SERVICE_NAME USERNAME
+
+   **Example:**
+
+   .. code-block:: bash
+
+       keyring set MSCOLAB_AUTH_{url} user@example.com
```

Comment:
that is for an auth protection, when you setup an mscolab_auth.py and have this in your search path.

In that case it is as username not user@example.com. We use a name, in the default config it is just mscolab

using this as an example is then 
`keyring set MSCOLAB_AUTH_http://localhost:8083 mscolab`